### DISCUSSION TITLE
Survivorship free data handling

### DISCUSSION THREAD

**Discussion by ghost** [edtechre/pybroker#157] [Q&A] [answered]
Hi, how does the package handle delisted stocks, say I have a position open and the stock gets delisted before my strategy closes out, what happens?

Thanks
Mark

**Comment by edtechre** [comment, marked as answer]
Hi @markxbaker,

Your positions are only updated on OHLC bars in the data source. If a stock is delisted, and there are no longer OHLC bars, then your position will stay open by default.  You can use [StrategyConfig#exit_on_last_bar](https://www.pybroker.com/en/latest/reference/pybroker.config.html#pybroker.config.StrategyConfig.exit_on_last_bar) to automatically close out the position on the last bar of data, which will cause the position to be closed if its stock is delisted.

**Reply by ghost** [reply]
Thanks Ed, loving the package by the way now I'm more up to speed, is pretty awesome!
